HHS/OPRE-OPS
# Ops 1232 agreement filter button 2

## What changed

Added filter functionality to Agreements List page.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1232

## How to test

Navigate to the Agreements List page and click on the Filters button.

Interact with the various filters, tags, and buttons to verify it works as expected.

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/1972158/1988952d-01d2-4b11-8bb4-0d8e68d1b262)
